tutorials/pick_and_place/ROS/src/niryo_moveit/scripts/mover.py

```
@@ -87,28 +87,41 @@ def plan_pick_and_place(req):
 
     # Pre grasp - position gripper directly above target object
     pre_grasp_pose = plan_trajectory(move_group, req.pick_pose, current_robot_joint_configuration)
+    
+    # If the trajectory has no points, planning has failed and we return an empty response
```

Will the empty response hide the failure to find a plan so that customers will not know Moveit failed to find a plan?
If so, is it possible to add some loggings for the failure to find a plan?

---

> Will the empty response hide the failure to find a plan so that customers will not know Moveit failed to find a plan?
No - I'm not logging an error here because the failure is logged by the trajectory planner when it fails to find. MoveIt also will report an empty response, and Unity will log that no response was received.